src/app/action-items/action-items.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="table-container" role="table" aria-label="Tasks">
+    <div class="flex-table header" role="rowgroup">
+        <div class="flex-row" role="columnheader">Title</div>
+        <div class="flex-row" role="columnheader">Type</div>
+        <div class="flex-row" role="columnheader">Completed</div>
+        <div class="flex-row" role="columnheader">Due Date</div>
+        <div class="flex-row" role="columnheader">Days Left</div>
+    </div>
+
+    <div class="flex-table row" role="rowgroup" *ngFor="let t of dataSource">
+        <div class="flex-row body-cell" role="cell">
+            {{ t.title }}
```

Comment:
Why this line is formatted differently than the lines below?

Reply:
This line improved, formatted as the other ones.